.prettierignore

```diff
@@ -40,3 +40,6 @@ yarn-error.log*
 .prettier*
 .editorconfig
 .gitignore
+
+# images
+packages/docs/img
```

Comment:
This is not a package, since we don't import it inside the JS files.

Let's create this in the root folder.